ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.10.yaml

```diff
@@ -117,6 +117,7 @@ resources:
       memory: 200Mi
 tests:
 - as: ocp-e2e-aws-arm64
+  cluster: arm01
```

Comment:
This can be removed.

Reply:
I did some digging. You could keep this field, but it is really strange why the validation is failing since we include arm01 in the validation. I am guessing that the `ci-operator` image that exists in the `arm01` cluster, is older version. Can you confirm this?